packit.yml

```diff
@@ -0,0 +1,19 @@
+# Enable RPM builds and running integration tests in PRs through https://packit.dev/
+# To use this, enable Packit-as-a-service in GitHub: https://packit.dev/docs/packit-as-a-service/
+# See https://packit.dev/docs/configuration/ for the format of this file
+
+specfile_path: cockpit-certificates.spec
+actions:
+  post-upstream-clone: make cockpit-certificates.spec
+  # reduce memory consumption of webpack in sandcastle container
+  # https://github.com/packit/sandcastle/pull/92
+  # https://medium.com/the-node-js-collection/node-js-memory-management-in-container-environments-7eb8409a74e8
+  create-archive: make NODE_OPTIONS=--max-old-space-size=500 dist-gzip
+  # certificates.git has no release tags; your project can drop this once you have a release
+  get-current-version: make print-version
+jobs:
+  - job: tests
+    trigger: pull_request
+    metadata:
+      targets:
+      - fedora-all
```

Comment:
I was not able to find info if I can still introduce a new package to fedora-33. I see however that we never tested this package against fedora-33, it's just fedora-34.

Some hint if we want to introduce it to f33 as well?

Reply:
Seems fine to me to start with fedora-34 and fedora-rawhide